nbformat/json_compat.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) IPython Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+try:
```

Comment:
i think we need one validator-of-last-resort as a hard `install_requires`, so would not make this dependency conditional.

Reply:
It is not, but that was the original code.